pkg/rbac/middleware.go

```diff
@@ -20,34 +20,33 @@ func Middleware() func(http.Handler) http.Handler {
 			cookie, err := r.Cookie("acm-access-token-cookie")
 			if err == nil {
 				clientToken = cookie.Value
-				klog.Info("Got user token from Cookie.")
+				klog.V(5).Info("Got user token from Cookie.")
 			} else if r.Header.Get("Authorization") != "" {
-				klog.Info("Got user token from Authorization header.")
+				klog.V(5).Info("Got user token from Authorization header.")
 				clientToken = r.Header.Get("Authorization")
-			} else {
-				http.Error(w, "Could not find a valid token.", http.StatusUnauthorized)
-				return
 			}
-			//Retrieving and verifying the token:
+			// Retrieving and verifying the token
 			if clientToken == "" {
-				http.Error(w, "Could not find client token", http.StatusUnauthorized)
+				klog.V(4).Info("Request didn't have a valid authentication token.")
+				http.Error(w, "{\"message\":\"Request didn't have a valid authentication token.\"}", http.StatusUnauthorized)
```

Comment:
- The first message is logged in our search-api logs.
- Second line is sent to the client. Changed to JSON so that it displays nicely on the Playground UI.

<img width="1913" alt="image" src="https://user-images.githubusercontent.com/4671325/166327185-18c39e7c-df0d-4702-a57b-b04afe81fc54.png">